### ISSUE TITLE
Clarification on loss weights

### ISSUE THREAD

**Issue by spcarney34** [lululxvi/deepxde#982]
Hello, 

Just wanted to clarify something (sorry, I'm a noob) -- if our loss is: 

$$ \mathcal{L} = \frac{1}{N_{\rm pde}} \sum_{i=1}^{N_{\rm pde}} \Big( \textrm{PDE residual} \Big) + \frac{1}{N_{\rm bdry}} \sum_{i=1}^{N_{\rm bdry}} \Big( \textrm{boundary residual} \Big) $$

and we manually set 
`model.compile("adam", lr=1e-3, loss_weights = [c1, c2])`

for some scalars $c_1$ and $c_2$, does specifying these loss-weights imply that our loss function is 

$$ \mathcal{L} = \frac{c_1}{N_{\rm pde}} \sum_{i=1}^{N_{\rm pde}} \Big( \textrm{PDE residual} \Big) + \frac{c_2}{N_{\rm bdry}} \sum_{i=1}^{N_{\rm bdry}} \Big( \textrm{boundary residual} \Big) $$

or 

$$ \mathcal{L} =  c_1 \sum_{i=1}^{N_{\rm pde}} \Big( \textrm{PDE residual} \Big) + c_2 \sum_{i=1}^{N_{\rm bdry}} \Big( \textrm{boundary residual} \Big) $$

Thanks in advance for the response. 
Best,
--Sean

**Comment by lululxvi** [comment]
The first.

**Comment by spcarney34** [comment]
Great, thanks @lululxvi. And thanks also for this great software and its maintenance.
